airbyte-integrations/connectors/source-stripe/unit_tests/integration/test_external_account_cards.py

```diff
@@ -211,7 +212,7 @@ def test_given_no_state_when_read_then_use_external_accounts_endpoint(self, http
         output = self._read(_config().with_start_date(_A_START_DATE), _NO_STATE)
         most_recent_state = output.most_recent_state
         assert most_recent_state.stream_descriptor == StreamDescriptor(name=_STREAM_NAME)
-        assert most_recent_state.stream_state == AirbyteStateBlob(updated=int(_NOW.timestamp()))
+        assert int(most_recent_state.stream_state.updated) == int(_NOW.timestamp())
```

Comment:
Does that means that the format of the state has changed from `int` to `str`? Does that means that we can't revert this change? We should at least make sure to be explicit on `Can this PR be safely reverted and rolled back?` in the PR description

Reply:
> Does that means that the format of the state has changed from int to str?
Yes, the low-code CDK stores states as a string. 

> Does that means that we can't revert this change?
I don't think so, but I need to double-check to be 100% sure.